src/domains/transaction/pages/SendTransfer/SendTransfer.test.tsx

```diff
@@ -510,7 +510,7 @@ describe("SendTransfer", () => {
 						.signatory()
 						.multiSignature({
 							min: 2,
-							publicKeys: [wallet.publicKey()!, profile.wallets().last().publicKey()!],
+							publicKeys: [wallet.publicKey()!, secondWallet.publicKey()!],
```

Comment:
The last wallet in profile might not have a public key, that is why it was throwing an error and failing